### PR TITLE
Get reactions with all components

### PR DESCRIPTION
**Pull request by dangunter**
## Fixes/Addresses:

#164

## Changes proposed in this PR:
- [x] Move old behavior behind non-default flag value
- [x] Add new behavior
- [x] Add some tests to make sure it works

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by dangunter** [comment]
@aladshaw3 I think I have something working. I am not happy with the implementation, but that's life with databases, sometimes you discover after you've done a few queries you need to structure the input a bit better. It should be performant enough for now, and when I fix the implementation it should have no impact on the rest of the code.

File: proteuslib/edb/tests/test_db_api.py [hunk 1]
```diff
@@ -72,3 +78,90 @@ def test_edb_load_convention():
...
+    "equilibrium_constant": "van_t_hoff_aqueous",
+    "equilibrium_form": "log_power_law",
+    "concentration_form": "ConcentrationForm.molarity",
+    "parameter_data": {
+        "dh_rxn_ref": [{"v": 55.83, "u": "kJ/mol", "i": 0}],
+        "ds_rxn_ref": [{"v": -80.7, "u": "J/mol/K", "i": 0}],
+    },
+    "type": "equilibrium",
+    "name": "H2O_Kw",
+    "components": ["H2O", "Kw"],
```
Comment (aladshaw3) [inline proteuslib/edb/tests/test_db_api.py:117]: I am a little confused by what the "components" is in this structure, but I don't think it matters for the time being.

**Review by aladshaw3 — APPROVED** [review]
Looks good to me.

File: proteuslib/edb/tests/test_db_api.py (hunk 1 above)
Comment (dangunter) [reply proteuslib/edb/tests/test_db_api.py:117]: that's a bug. components is generated by splitting the name and Kw looked like the name of an element to the regular expression

**Review by TimBartholomew — APPROVED** [review]
LGTM

**Pull request merged**